journals/apps/api/v1/content/views.py

```diff
@@ -0,0 +1,22 @@
+"""
+Overridden Wagtail API endpoints
+"""
+from rest_framework.permissions import AllowAny
+from rest_framework.exceptions import PermissionDenied
+from wagtail.api.v2.endpoints import PagesAPIEndpoint
+from wagtail.wagtailcore.models import Page
+# from wagtail.wagtailcore.utils import resolve_model_string
+
+from journals.apps.journals.models import JournalAccess, JournalPage
+from journals.apps.api.filters import PageAuthorizationFilter
+
+class FilteredPagesAPIEndpoint(PagesAPIEndpoint):
+    """
+    Filters out some page types if the requesting user doesn't have access to them.
+
+    We need to return JournalIndexPage and JournalAboutPage content types to everyone (including anonymous).
+    We also need to check a user's access to a Journal before returning any JournalPage attached to that journal.
+    """
+
+    permission_classes = (AllowAny,)
```

Comment:
maybe change this from `AllowAny` to `IsAuthenticated` and then you can remove your check for is_authenticated in the filter

Reply:
Can't do that because IsAuthenticated would block anonymous users from viewing the about and index pages.